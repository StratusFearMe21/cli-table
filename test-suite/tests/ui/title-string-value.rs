//! `title` only accepts `&str` values
use cli_table::Table;

#[derive(Table)]
struct Test {
    #[table(title = 1)]
    a: u8,
}

fn main() {}
