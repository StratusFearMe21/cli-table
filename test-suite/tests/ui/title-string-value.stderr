error: Invalid value for #[table(title = "field_name")]
 --> $DIR/title-string-value.rs:6:21
  |
6 |     #[table(title = 1)]
  |                     ^
